packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -1,19 +1,42 @@
-import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
+import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { NavItemSlots, NavItemState } from './NavItem.types';
-import { typographyStyles } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
 
 export const navItemClassNames: SlotClassNames<NavItemSlots> = {
   root: 'fui-NavItem',
   content: 'fui-NavItem__content',
+  icon: 'fui-NavItem__icon',
+};
+
+const navItemTokens = {
+  indicatorOffset: '18',
+  indicatorWidth: '4',
+  indicatorHeight: '20',
```

Comment:
since you are removing the px part of the string, we could just have it be a number and could be used in the future to leave calculations out of css.
```suggestion
  indicatorOffset: 18,
  indicatorWidth: 4,
  indicatorHeight: 20,
```

Reply:
updated -thanks!